README.md

```diff
@@ -46,6 +46,11 @@ If this is not set, you may see errors with the message:
 
 `Could not resolve substitution to a value: ${DB_PORT}`
 
+#### Removing tests
+
+If an API test is removed because it is no longer needed, you will need to modify the list of required status checks on Github. These can be found in "Settings > Branches" under the `master` branch. The branch status checks are run when a pull request is raised and while get stuck if it tries to run a check on a test that is no longer present. You can remove the test at this point and the checks will carry on to completion.   
```

Comment:
Could you put line breaks in at the following places:

- after `... checks on Github.` and before `These can be found ...`
- after `... master branch.` and before `The branch status checks ...`

Slight re-wording: `... is raised and will get stuck ...`